backend/SlothOrganizer/SlothOrganizer.Persistence/Properties/Dashboards/InsertDashboard.sql

```diff
@@ -0,0 +1,4 @@
+IF EXISTS (SELECT * FROM Users WHERE Id = @UserId)
```

Comment:
To clarify: you add if user exists. In such case if he is not, your select returns null. Dashbord model in repository use QuerySingleAsync. So if user does not exists you will get an exception that is not handled. That is not ok. If you really expect for some reason that there is a possibility to create a dashboard with not existing user, than it must be handled, error for user should be shown, etc. But it is still not clear why you expect that user will not exist.

Reply:
Removed if exists